xchange-core/src/main/java/com/xeiam/xchange/dto/account/AccountInfo.java

```diff
@@ -71,6 +72,21 @@ public BigDecimal getTradingFee() {
   }
 
   /**
+   * Returns the wallet of the specified currency.
+   *
+   * @param currency one of the {@link Currencies}.
+   * @return the wallet of the specified currency, or a zero balance wallet if no wallet with such currency.
+   */
+  public Wallet getWallet(String currency) {
```

Comment:
You could keep the wallets as a Map, and change `getWallets()` to `{new ArrayList(wallets.values());}` instead